.github/ISSUE_TEMPLATE/bug.yml

```
@@ -120,6 +120,17 @@ body:
       label: Windows Subsystem for Linux
       description: If you are running VS Code on Windows Subsystem for Linux (WSL), please specify what version of WSL and Windows you are using.
       placeholder: e.g. WSL 2, Windows 11 21H2
+  - type: textarea
+    id: bug-reproduction
```

[suggegst] If just a URL is needed, is `input` easier to enter?

```suggestion
  - type: input
    id: reproduction-url
```